hazelcast-spring/src/main/resources/hazelcast-spring-5.2.xsd

```diff
@@ -1534,47 +1535,45 @@
         </xs:complexType>
     </xs:element>
 
-    <xs:complexType name="registered-classes">
+    <xs:complexType name="compact-serializers">
```

Comment:
This is just a complex type definition, which is used in `compact-serialization`. 

It is not named as `serializers` as the type is defined separately, to avoid possible confusion.

Note that, the name of the complex type has nothing to do with how it is used. The user provides it as `<serializers></serializers>` regardless of its complex type name